### DISCUSSION TITLE
remember_state & FRIENDLY_NAME/set

### DISCUSSION THREAD

**Discussion by pasaone** [Koenkk/zigbee2mqtt#12876] [Devices]
I'm brand new to zigbee2mqtt and I'm stuck on adding a feature to a device. Also I'm not on HomeAssistant but via Homebridge only

On Osram lights, there is the ability to add to FRIENDLY_NAME/set to keep the state of the last light position in case of power failure

The value is 
{
    "remember_state: true
}

And now I'm lost and I don't know where to add it...
In the frontend?
In the configuration.yaml file?
Via something else?

And there is a note
NOTE: This must be executed every time you make changes to a light's attributes for it to then 'remember'.

Thanks for your help

**Comment by sk8er000** [comment]
I'm stuck too I think I have to edit the state.json but I'm not sure and I don't wanto to break the extension

**Comment by danielocdh** [comment]
it didn't work for my [OSRAM AA70155](https://www.zigbee2mqtt.io/devices/AA70155.html) but reading [this](https://www.zigbee2mqtt.io/guide/usage/mqtt_topics_and_messages.html#publishing-messages) I had to run this on pi terminal: `mosquitto_pub -t 'zigbee2mqtt/room_name/set' -m '{"remember_state":true}'`, then the debug log on zigbee2mqtt said: `Publishing 'set' 'remember_state' to 'room_name'`. `room_name` is a group that contains the light.

The chance of this feature working is why I was trying zigbee2mqtt, I can set the lights to "remember" on deCONZ but it is buggy/useless too
